build/paths.js

```diff
@@ -36,4 +38,21 @@ paths.js.root             = path.join(paths.root,'src','js')
 paths.rust                = {}
 paths.rust.root           = path.join(paths.root,'src','rust')
 
+function project_manager_path(root) {
+    let base_path = path.join(root, 'bin')
+    const target_platform = os.platform()
+    switch (target_platform) {
+        case 'linux':
+            return path.join(base_path, 'project-manager')
+        case 'darwin':
+            return path.join(base_path, 'project-manager')
+        case 'win32':
+            return path.join(base_path, 'project-manager.exe')
+        default:
+            throw 'UnsupportedPlatform: ' + target_platform
+    }
+}
+
+paths.get_project_manager_path = project_manager_path
```

Comment:
if you want to do something liekthat in JS you can always write `paths.foo = function() { ... }` not to repeat the name several times.